model/distro/distro.go

```diff
@@ -63,6 +63,16 @@ const (
 	BootstrapMethodSSH                = "ssh"
 	BootstrapMethodPreconfiguredImage = "preconfigured-image"
 	BootstrapMethodUserData           = "user-data"
+
+	// Recognized architectures, should be in the form ${GOOS}_${GOARCH}.
+	ArchDarwinAmd64  = "darwin_amd64"
```

Comment:
I think it'd be nice to have this list be sent to the client on the distros page, so that this is the only list of agent architectures (rather than maintaining an identical list in the UI)

Reply:
Unfortunately the way the distros page was written, all the distro options (including scheduler version, provider, and bootstrap method) have this disconnect between the JavaScript validation and server validation. It would take a non-trivial amount of effort to overhaul the distros page.

(I would also like this, but not in this ticket.)